### DISCUSSION TITLE
esx_banking

### DISCUSSION THREAD

**Discussion by BMOREGTA** [community/community#59949] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How do I set up this banking script? and does it still work? I tried it but it doesn't work I put the SQL file in my database dropped the folder in resource under addon esx and the name in server.cfg I took out my original bank fleeka which was just a ATM deposit and withdraw that's it and then I put in esx_banking and its not working is there anything I need to do in the config or any file to start it up?

**Comment by otter-computer** [comment]
Hi @BMOREGTA,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the project's repository. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
